serpentTools/parsers/__init__.py

```diff
@@ -45,7 +47,8 @@
     r'(.*_det\d+\.m)': DetectorReader,
     r'(.*_res\.m)': ResultsReader,
     r'(.*_fmtx\d+\.m)': FissionMatrixReader,
-    r'(.*\.bumat\d+)': BumatReader
+    r'(.*\.bumat\d+)': BumatReader,
+    r'(.*_xs[0-9]*\.m)' : XSPlotReader
```

Comment:
Minor tweak: replace `[0-9]*` with `\d*` for the same effect with one less character lol

Reply:
All my life, I've used [0-9] and not \d. Thanks for the enlightenment.